include/rtc.h

```diff
@@ -15,7 +15,11 @@ extern "C" {
 #endif
 
 /** Number of RTC ticks in a second */
+#ifdef CONFIG_RTC_PRESCALER
```

Comment:
I couldn't find RTC_PRESCALER option in Kconfig files.

Reply:
It is only defined by RTC driver implementations and not really applicable for this hardware. There is no direct access to the RTC counter on STM32, so the driver operates in milliseconds and we directly define RTC_ALARM_SECOND to be used as time base.

Reply:
The description of RTC_ALARM_SECOND is the same as the prescaler property in DT:
`RTC prescaler used to determine ticks per second`. It is confusing.

Reply:
I will revert the change in `rtc.h`, but please see below.

Reply:
Actually, `#define RTC_ALARM_SECOND (32768 / CONFIG_RTC_PRESCALER)` and we need RTC_ALARM_SECOND to equal 1000, so it's not the same. CONFIG_RTC_PRESCALER would need to equal 32.768

Reply:
I wasn't clear. 
```
config RTC_ALARM_SECOND
	int
	default 1000
	prompt "Number of RTC ticks per second"
	help
	  RTC prescaler used to determine ticks per second
```
The prompt and help section of `RTC_ALARM_SECOND` option should be updated. They are confusing because they match the description of the prescaler property in rtc.yaml.
```
    prescaler:
      type: int
      category: required
      description: Prescaler value denotes number of RTC ticks in 1 second
      generation: define
```